### ISSUE TITLE
Install Trac GitHub pluging

### ISSUE THREAD

**Issue by adiroiban** [twisted-infra/braid#168]
https://github.com/trac-hacks/trac-GitHub

it should be done first in the staging trac, but it should be done via braid so that we can also push the changes later to production

**Comment by hawkowl** [comment]
Done, in https://github.com/twisted-infra/braid/tree/trac-github

**Comment by adiroiban** [comment]
should this be close before having that code in master?

**Comment by hawkowl** [comment]
I think so. The master ticket will be the one closed when all of this is merged into master and put into production.
